src/transformers/configuration_auto.py

```diff
@@ -126,6 +129,7 @@
         ("longformer", "Longformer"),
         ("roberta", "RoBERTa"),
         ("flaubert", "FlauBERT"),
+        ("fsmt", "FSMT"),
```

Comment:
Let's put the full name here as this dictionary is used for docstrings, so "FairSeq Machine-Translation".